src/index.js

```diff
@@ -0,0 +1,265 @@
+import { setDate } from './modules/helpers';
+// eslint-disable-next-line
+import {
+  Project, projectsToDom, deleteProject, checkProject,
+} from './modules/projects';
+// eslint-disable-next-line
+import { makeTaskForm, makeTaskSecOnClck, changeStatus, Task, makeTaskSection, makeEditForm, deleteAllTasks, countForm, makeToday,
+} from './modules/tasks';
+
+const addForm = document.getElementById('addForm');
+const getName = document.getElementById('nameInput');
+const getDescription = document.getElementById('descriptionInput');
+const first = document.getElementById('first');
+const projectsContainer = document.getElementById('projectsContainer');
+const tableArticle = document.getElementById('tableArticle');
+const editForm = document.getElementById('editForm');
+const editTaskForm = document.getElementById('editTaskForm');
+
+// eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files